scripts/update_terminfo.bash

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
`.sh` should be the filename extension, we don't have `.bash`, that just adds entropy.